view/template/acquisition/_youtube_error.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+$error_message = AcquisitionActions::prepareYoutubeErrorMessage($error_message ,($email ?? null));
```

Comment:
@maximest-pierre you're still not understanding how variables get into view partials. Will discuss with you today. It should never be necessary to access a controller function directly from a view. In general the only PHP that should be appearing in templates is basic control statements (if / for) and printing variables.